sklearn/decomposition/fastica_.py

```diff
@@ -431,7 +348,10 @@ def my_g(x):
     Attributes
     ----------
     components_ : 2D array, shape (n_components, n_features)
-        The unmixing matrix.
+        The linear operator to apply to the data to get the independent
+        sources. This is equal to the unmixing matrix when ``whiten`` is
+        False, and equal to ``np.dot(unmixing_matrix, whitening_)`` when
```

Comment:
```suggestion
        False, and equal to ``np.dot(unmixing_matrix_, whitening_)`` when
```

Reply:
The unmixing matrix is a private attribute